browser.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
+/* globals alert */
+
+const semver = require('semver')
+
+const getInstalledVersion = require('./lib/check-node')
+const loadVersions = require('./lib/load.js')
+// const install = require('./lib/install')
+// for testing purpose
```

Comment:
@addaleax nope, I leaved there only for testing, in this way if one of you wants to try my proposal can do it without installing a lot of times Node :P
In addition as writed above I have to implement the error management.
I've opened the pull request so I can get suggestions and improvements before finish all the work :)